# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1493954736 14400
#      Thu May 04 23:25:36 2017 -0400
# Node ID 7308cdea0c2b2e3cc72ff70485aa7a0216e9fb2a
# Parent  a41fdb801db67eb8e5cf4c11b484a28963aa579d
* pt-decl.cc: Fix error in previous change.

diff --git a/libinterp/parse-tree/pt-decl.cc b/libinterp/parse-tree/pt-decl.cc
--- a/libinterp/parse-tree/pt-decl.cc
+++ b/libinterp/parse-tree/pt-decl.cc
@@ -77,17 +77,18 @@ namespace octave
   {
     if (t)
       {
         if (cmd_name == "global")
           mark_global ();
         else if (cmd_name == "persistent")
           mark_persistent ();
         else
-          error ("tree_decl_command: unknown decl type: %s", cmd_name);
+          error ("tree_decl_command: unknown decl type: %s",
+                 cmd_name.c_str ());
       }
   }
 
   tree_decl_command::~tree_decl_command (void)
   {
     delete init_list;
   }
 
